hooks/useAvalancheForecastFragment.ts

```diff
@@ -15,13 +19,14 @@ export const useAvalancheForecastFragment = (center_id: AvalancheCenterID, forec
     },
     {
       enabled: !!fragments,
-      staleTime: 60 * 60 * 1000, // re-fetch in the background once an hour (in milliseconds)
-      cacheTime: 24 * 60 * 60 * 1000, // hold on to this cached data for a day (in milliseconds)
+      staleTime: options?.skipCacheForTests ? 0 : 60 * 60 * 1000, // re-fetch in the background once an hour (in milliseconds)
```

Comment:
Are we certain there is no approach that does not require this? IMHO production code with conditionals for testing is a huge anti-pattern.

Reply:
I think the alternative is to make callers always pass an explicit cache time. This feels like a reasonable tradeoff to me but open to alternatives 🤷 

It turns out the way cache expiration is implemented in React Query is via `setTimeout`. Jest will wait for all pending async tasks to complete if you have a lingering `setTimeout`, and then the test itself will eventually timeout after 5 seconds (Jest's default test timeout) and fail.